arch/risc-v/src/hpmicro/hpm6750/Make.defs

```diff
@@ -0,0 +1,28 @@
+############################################################################
+# arch/risc-v/src/hpm6750/Make.defs
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+include common/Make.defs
+
+# Specify our C code within this directory to be included
+CHIP_CSRCS = hpmicro$(DELIM)hpm6750$(DELIM)hpm_allocateheap.c hpmicro$(DELIM)hpm6750$(DELIM)hpm_clockconfig.c
+CHIP_CSRCS += hpmicro$(DELIM)hpm6750$(DELIM)hpm_irq.c hpmicro$(DELIM)hpm6750$(DELIM)hpm_irq_dispatch.c
+CHIP_CSRCS += hpmicro$(DELIM)hpm6750$(DELIM)hpm_lowputc.c hpmicro$(DELIM)hpm6750$(DELIM)hpm_serial.c
+CHIP_CSRCS += hpmicro$(DELIM)hpm6750$(DELIM)hpm_start.c hpmicro$(DELIM)hpm6750$(DELIM)hpm_timerisr.c
+CHIP_CSRCS += hpmicro$(DELIM)hpm6750$(DELIM)hpm_printf.c
```

Comment:
You don't need to pass the entire patch, all file are already inside your hpm6750/
Please look how it is done in other RISC-V chip like arch/risc-v/src/esp32c3/Make.defs